### ISSUE TITLE
Can't use WebP on Heroku

### ISSUE THREAD

**Issue by Filpin011** [python-pillow/Pillow#7366]
### What did you do?
Saving an image webp to buffer, converting from jpeg or png

### What did you expect to happen?
No error locally and on others deployments

### What actually happened?

Hi, i convert from base64 image to webp, locally all works fine, also in fly.io deployment, but when i moved to heroku i cant do the same things that i do elsewhere.
I post part of my code and a screenshot of the error

### What are your OS, Python and Pillow versions?

* OS: 5.15.0-1041-aws (heroku docker)
* Python: 3.11.5
* Pillow: 9.0.1

```python

img=Image.open(io.BytesIO(base64.decodebytes(bytes(myimage, "utf-8"))))
    img=ImageOps.exif_transpose(img)
    width, height = img.size
    TARGET_WIDTH = 500
    coefficient = width / 500
    new_height = height / coefficient
    img = img.resize((int(TARGET_WIDTH),int(new_height)),Image.ANTIALIAS)
    thumb_io = io.BytesIO() 
    img.save(thumb_io, 'webp', optimize=True, quality=85)
    uploaded_link=await upload_to_imgbb(thumb_io.getbuffer(),str(doguuid))
    img.close()
    thumb_io.close()

```

This is an image of the issue ![image](https://github.com/python-pillow/Pillow/assets/64793842/b8803d59-8d10-4061-985c-2f97379344e8)

**Comment by radarhere** [comment]
I expect you are missing the libwebp dependency - https://pillow.readthedocs.io/en/stable/installation.html#external-libraries

from PIL import features
features.check_module('webp')

How did you install Pillow?

**Comment by Filpin011** [comment]
> I expect you are missing the libwebp dependency - https://pillow.readthedocs.io/en/stable/installation.html#external-libraries
> 
> from PIL import features
> features.check_module('webp')
> 
> How did you install Pillow?

Hi, i do the check and libwebp is missing (i checked with the import ) but there is no way to install it with atp cause heroku dont allow me that i tried also this for install the dependency with dockerfile:

**Comment by radarhere** [comment]
What is the image that you're starting FROM with the Dockerfile? Are you able to post a complete Dockerfile so that we can replicate the problem without Heroku?

**Comment by Filpin011** [comment]
> What is the image that you're starting FROM with the Dockerfile? Are you able to post a complete Dockerfile so that we can replicate the problem without Heroku?

Hi, ill post it, but i think that heroku dont allow to execute apt-get, ill try with a buildpack that seems to allow that, ill update if that works

```

FROM python:${PYTHON_VERSION}

RUN apt-get update && apt-get install -y \
    python3-pip \
    python3-venv \
    python3-dev \
    python3-setuptools \
    python3-wheel \
    libwebp-dev

RUN mkdir -p /app
WORKDIR /app

COPY requirements.txt .
RUN pip install -r requirements.txt

COPY . .

EXPOSE 8080

CMD ["uvicorn", "main:app", "--host", "0.0.0.0", "--port", "8080"]

```

**Comment by Filpin011** [comment]
> What is the image that you're starting FROM with the Dockerfile? Are you able to post a complete Dockerfile so that we can replicate the problem without Heroku?

Ok i have an update, i had installed libwebp with apt but it doesn't fix the problem :( 

Idk why that happens, and when I check the module webp it give me false as result

![image](https://github.com/python-pillow/Pillow/assets/64793842/4435f983-fb56-4c73-8987-bba2d8da909f)

**Comment by radarhere** [comment]
I attempted to replicate this using your Dockerfile, but it works when I run it on GitHub Actions - https://github.com/radarhere/docker-images/commit/0d7f244a4a0d297a7be8026ebb9674c210a56161 / https://github.com/radarhere/docker-images/actions/runs/6064563343/job/16453134836#step:6:216

**Comment by Filpin011** [comment]
> I attempted to replicate this using your Dockerfile, but it works when I run it on GitHub Actions - https://github.com/radarhere/docker-images/commit/0d7f244a4a0d297a7be8026ebb9674c210a56161 / https://github.com/radarhere/docker-images/actions/runs/6064563343/job/16453134836#step:6:216

I tried the same on render.com, and this dockerfile works, but i would prefer use heroku, but i cant figure out :( all the times same error

**Comment by Filpin011** [comment]
I fixed by upgrading the Pillow version to 9.3.0 the first with python 3.11 support, but i cant understand why on other platform worked with the older version, ill close this

**Comment by homm** [comment]
[Pillow 9.2.0](https://pypi.org/project/Pillow/9.2.0/) and later has precompiled binaries for python 3.11.
